docs/content/concepts/assets/asset-auto-execution.mdx

```diff
@@ -121,6 +122,27 @@ def asset1(upstream1, upstream2):
     ...
 ```
 
+#### Auto-materialize on a regular cadence
+
+By default, the eager policy will only materialize an asset if it's missing or one of its parents update. This means that an unpartitioned root asset will only get auto-materialized a single time. In some cases, it's desirable to recompute these assets on a regular basis. To enable this, the `materialize_on_cron` rule can be added to a given policy.
```

Comment:
The main use case for this is a root asset, right? Should we focus the example on that and communicate this to users in some way? It seems potentially easy for them to shoot themselves in the foot by adding it to a non-root asset.